packages/client3/components/design-system/avatar/avatar.tsx

```diff
@@ -0,0 +1,47 @@
+
+export function Avatar({ image, link, size = 11 }: AvatarProps) {
+  const hrefAttribute = link || "";
+  const sizeClass = `w-${size} h-${size}`;
+  return (
+    <div className="flex w-max flex-row items-center">
+      <div
+        className={clsx(
+          "relative overflow-hidden rounded-full bg-dark-50 dark:bg-dark-90",
+          sizeClass
+        )}
+      >
+        <Link href={hrefAttribute} className="!no-underline">
+          <Image
+            src={image ? image : placeHolder}
```

Comment:
instead of placeHolder, I would say make it `default`

Reply:
`default` is a reserved name, we can't use it to create a variable. 
`'default' is not allowed as a variable declaration name.ts(1389)`

Reply:
`defaultImage` then?